charts/ziti-controller/templates/configmap.yaml

```diff
@@ -273,7 +273,7 @@ data:
             # address - required
             # The public address that external incoming requests will be able to resolve. Used in request processing and
             # response content that requires full host:port/path addresses.
-            address: {{ .Values.managementApi.advertisedHost | default (printf "%s-mgmt.%s.svc.cluster.local" .Release.Name .Release.Namespace) }}:{{ .Values.managementApi.advertisedPort }}
+            address: {{ .Values.managementApi.advertisedHost | default printf "%s-ctrl.%s.svc.%s" .Release.Name .Release.Namespace .Values.ca.clusterDomain) }}:{{ .Values.managementApi.advertisedPort }}
```

Comment:
The cluster domain part is fine, but `-mgmt` is the conventional suffix for the management API when it is configured separately from the client API, i.e., when `managementApi.service.enabled` is true.

The `-ctrl` conventional suffix applies to the ctrl plane which is a different server provided by the controller for routers.